packages/babel-preset-env/data/built-in-modules.json

```diff
@@ -6,7 +6,7 @@
     "safari": "10.1",
     "opera": "48",
     "ios_saf": "10.3",
-    "and_chr": "71",
-    "and_ff": "64"
+    "and_chr": "74",
```

Comment:
I don't think we need to bump `and_chr` and `and_ff` because:
1) CanIUse doesn't support mobile browsers versions
2) `caniuse-db` is not bumped

Reply:
> CanIUse doesn't support mobile browsers versions

What do you mean? This data is generated directly via our [script](https://github.com/babel/babel/blob/master/packages/babel-preset-env/scripts/build-modules-support.js)?

![image](https://user-images.githubusercontent.com/56288/69273798-af2e3200-0b9e-11ea-8a5d-5c179f12e0c5.png)

> `caniuse-db` is not bumped

FWIW, if you run `build-data` on master, it updates these two values with the version of `caniuse-db` that it's pinned to. In fact, if we bump it, it'll update `and_chr` to 78.

Reply:
They should not be updated because caniuse does not track old mobile browser versions, so `and_chr` will always return the latest version of all available Android Chrome, same as `and_ff`. (i.e. they are bumped in this [commit](https://github.com/Fyrd/caniuse/commit/3bc36b569c7506b38d34c4a2da07ed7794436ec5) of caniuse-db)

Could we skip `and_ff` and `and_chr` in https://github.com/babel/babel/blob/fee4d936a3443c4d8812cac208e567e0ba8edc9d/packages/babel-preset-env/scripts/build-modules-support.js#L6

Reply:
Oh, gotcha. Yep I'll make an update.

Reply:
Related: https://github.com/babel/babel/pull/9978/files

Reply:
Hard to keep everything in your head 🤦‍♂

Reply:
(So true. `git-blame` is my very best reference for code review.)

Reply:
According to https://github.com/babel/babel/pull/7344#issue-167553342, it was you who explained it to me :stuck_out_tongue:

Reply:
@nicolo-ribaudo Any idea why `and_chr` is removed from `skipList` in #9978?

Reply:
Adding them to `skipList` would remove them from the output altogether

Reply:
Yeah, and it doesn't really matter since browserlists returns the desktop version anyway.

Reply:
Then we need a `doNotUpdateList` then. Anyway let's address it in a separate PR.